pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,22 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	// +optional
+	Config *PrometheusConfiguration `json:"config,omitempty"`
+}
+
+// Config  is the the desired state of Prometheus Configuration
+type PrometheusConfiguration struct {
+	ConfigMapRef *ConfigMapRef `json:"configMapRef,omitempty"`
+
+	CommandOptions []string `json:"command,omitempty" protobuf:"bytes,3,rep,name=commandOptions"`
```

Comment:
Remove 'protobuf:"bytes,3,rep,name=commandOptions' and add the comments to explain the usage of command Options.